.ci/kata-arch.sh

```diff
@@ -38,7 +38,8 @@ arch_to_golang()
 		aarch64) echo "arm64";;
 		ppc64le) echo "$arch";;
 		x86_64) echo "amd64";;
-		*) die "unsupported architecture: $arch";;
+        s390x) echo "s390x";;
```

Comment:
Please could you sort out the indent (I'm guessing it might be a "tabs vs. spaces" issue here?)

Reply:
I didn't set my vimrc properly just added `set autoindent noexpandtab tabstop=4 shiftwidth=4` Hopefully this should do the job